src/test/java/uk/co/ramp/covid/simulation/place/CareHomeTest.java

```diff
@@ -110,17 +110,22 @@ public void sickResidentsAreQuarantined() {
         Time t = new Time(0);
         while (!inf.getcVirus().isSymptomatic()) {
             inf.getcVirus().stepInfection(t);
+            inf.cStatus();
             t = t.advance();
         }
-        
+
         // Not quarantined instantly
+        // TODO You might have symptoms but have a trans adjustment of 0 (if you get symptom while latent).
```

Comment:
Should we add a GitHub issue to fix this new TODO?

Reply:
I might explore this a bit more today first (to fix, or have a better idea what goes in the issue). I'm not sure it's necessarily an issue with the "standard" COVID progression; it's possible that we need to do slightly more when we "force" symptomatic cases (only in the tests).

If I track this down I'll make another PR so Paul can double check, else I'll make an issue.